client/payment-details/dispute-details/style.scss

```
@@ -182,3 +182,17 @@
 		margin-bottom: 0;
 	}
 }
+
+.dispute-notice {
+	.dispute-notice-link {
+		display: block;
+	}
+}
+
+@media screen and ( max-width: $break-small ) {
+	.wcpay-inline-notice.components-notice
+		.components-notice__content
+		a.dispute-notice-link {
+		white-space: normal;
+	}
```

⛏️ for consistency, should this new classname follow BEM-naming?

```suggestion
	.dispute-notice__link {
		display: block;
	}
}

@media screen and ( max-width: $break-small ) {
	.wcpay-inline-notice.components-notice
		.components-notice__content
		a.dispute-notice__link {
		white-space: normal;
	}
```

---

Addressed in [13db0c1](https://github.com/Automattic/woocommerce-payments/pull/9790/commits/13db0c1a5a9f0c2336a9da3a796ed31e541c86d7).